pktinfo.c

```diff
@@ -12,6 +12,31 @@
 #include "pktinfo.h"
 #include "log.h"
 
+u64 ip4_rm_header(struct sk_buff *skb, unsigned int hdrlen){
+    struct iphdr *iph; 
+    struct tcphdr *tcp; 
+    u64 volume;
+    
+    iph = ip_hdr(skb);
+    volume = skb->len - hdrlen - iph->ihl * 4;
+
+    // Deduct the header length of transport layer
+    switch (iph->protocol) {
+		case IPPROTO_TCP: 
```

Comment:
Please indent and format source code based on Linux kernel coding style.

https://www.kernel.org/doc/html/latest/process/coding-style.html